lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -340,6 +340,26 @@ void MtohRenderOverride::UpdateRenderGlobals(
     MGlobal::executeCommandOnIdle("refresh -f");
 }
 
+VtValue MtohRenderOverride::_GetHdStGPUMemory() const
+{
+    if (_isUsingHdSt && _renderDelegate)
+    {
+        VtDictionary hdStRenderStat = _renderDelegate->GetRenderStats();
+        return hdStRenderStat[HdPerfTokens->gpuMemoryUsed.GetString()];
+    }
+    return VtValue();
+}
+
+int MtohRenderOverride::GetHdStGPUMemory()
+{   
+    int totalGPUMemory = 0;
+    std::lock_guard<std::mutex> lock(_allInstancesMutex);
+    for (auto* instance : _allInstances) {
+        totalGPUMemory += instance->_GetHdStGPUMemory().UncheckedGet<int>();
+    }
+    return totalGPUMemory; 
```

Comment:
I think Hydra reports this in bytes but it's not clear from the code. I could convert it to MB like in the code for CPU.